benchmarks/frameworks/torch/mpc/scripts/abstract/abstract_benchmark.py

```diff
@@ -0,0 +1,40 @@
+import abc
+
+from .workers_initialization import workers, hook
```

Comment:
Q: Could you use here the full path (only to be consistent with the rest of the files)

Reply:
Sure!